easybuild/tools/modules.py

```diff
@@ -997,12 +1015,18 @@ class EnvironmentModulesC(ModulesTool):
     """Interface to (C) environment modules (modulecmd)."""
     COMMAND = "modulecmd"
     REQ_VERSION = '3.2.10'
+    MAX_VERSION = '3.2.10'
```

Comment:
Make this a bit more liberal, in case people have local made local modifications and bumped this version.

```python
MAX_VERSION = '3.99'  # fictional version, just to indicate that version 4.0.0 and up are not compatible